modules/org/infinispan/distribution/jvm/added/bin/java-opts.sh

```diff
@@ -1,136 +1,59 @@
 if [ -n "${KSH_VERSION:-}" ]; then
   alias local=typeset
 fi
 
-# Generic formula evaluation based on awk
-calc() {
-  local formula="$1"
-  shift
-  echo "$@" | awk '
-    function ceil(x) {
-      return x % 1 ? int(x) + 1 : x
-    }
-    function log2(x) {
-      return log(x)/log(2)
-    }
-    function max2(x, y) {
-      return x > y ? x : y
-    }
-    function round(x) {
-      return int(x + 0.5)
-    }
-    {print '"int(${formula})"'}
-  '
-}
-
-max_memory() {
-  # High number which is the max limit until which memory is supposed to be
-  # unbounded.
-  local mem_file="/sys/fs/cgroup/memory.max"
-  if [ ! -r "${mem_file}" ]; then
-    mem_file="/sys/fs/cgroup/memory/memory.limit_in_bytes"
-  fi
-  if [ -r "${mem_file}" ]; then
-    local max_mem_cgroup="$(cat ${mem_file})"
-    local max_mem_meminfo_kb="$(cat /proc/meminfo | awk '/MemTotal/ {print $2}')"
-    local max_mem_meminfo="$(expr $max_mem_meminfo_kb \* 1024)"
-    if [ ${max_mem_cgroup:-0} != -1 ] && [ ${max_mem_cgroup:-0} -lt ${max_mem_meminfo:-0} ]
-    then
-      echo "${max_mem_cgroup}"
-    fi
-  fi
-}
+export JBOSS_CONTAINER_UTIL_LOGGING_MODULE="${JBOSS_CONTAINER_UTIL_LOGGING_MODULE-/opt/jboss/container/util/logging}"
+export JBOSS_CONTAINER_JAVA_RUN_MODULE="${JBOSS_CONTAINER_JAVA_RUN_MODULE-/opt/jboss/container/java/run}"
 
-init_limit_env_vars() {
-  # Read in container limits and export the as environment variables
-  local mem_limit="$(max_memory)"
-  if [ -n "${mem_limit}" ]; then
-    export CONTAINER_MAX_MEMORY="${mem_limit}"
-  fi
-}
-
-# ==========================================================================
-# Check for memory options and set max heap size if needed
-calc_max_memory() {
-  # Check whether -Xmx is already given in JAVA_OPTIONS
-  if echo "${JAVA_OPTIONS:-}" | grep -q -- "-Xmx"; then
-    return
-  fi
+# Default the application dir to the S2I deployment dir
+if [ -z "$JAVA_APP_DIR" ]
+  then JAVA_APP_DIR=/deployments
+fi
 
-  if [ -z "${CONTAINER_MAX_MEMORY:-}" ]; then
-    return
-  fi
+source "$JBOSS_CONTAINER_UTIL_LOGGING_MODULE/logging.sh"
 
-  # Check for the 'real memory size' and calculate Xmx from the ratio
-  if [ -n "${JAVA_MAX_MEM_RATIO:-}" ]; then
-    if [ "${JAVA_MAX_MEM_RATIO}" -eq 0 ]; then
-      # Explicitely switched off
-      return
-    fi
-    calc_mem_opt "${CONTAINER_MAX_MEMORY}" "${JAVA_MAX_MEM_RATIO}" "mx"
-  fi
-  # When JAVA_MAX_MEM_RATIO not set and JVM >= 10 no max_memory
-}
-
-# Check for memory options and set initial heap size if requested
-calc_init_memory() {
-  # Check whether -Xms is already given in JAVA_OPTIONS.
-  if echo "${JAVA_OPTIONS:-}" | grep -q -- "-Xms"; then
-    return
-  fi
+# ==========================================================
+# Generic run script for running arbitrary Java applications
+#
+# This has forked (and diverged) from:
+# at https://github.com/fabric8io/run-java-sh
+#
+# ==========================================================
```

Comment:
Is this still relevant or are we using https://github.com/jboss-container-images/openjdk/blob/ubi9/modules/run/artifacts/opt/jboss/container/java/run/run-java.sh ?